src/bridge/lib/wallet.js

```diff
@@ -5,7 +5,7 @@ import * as lodash from 'lodash';
 import Maybe from 'folktale/maybe';
 import * as secp256k1 from 'secp256k1';
 import * as ob from 'urbit-ob';
-import * as kg from '../../../node_modules/urbit-key-generation/dist/index';
+import * as kg from 'urbit-key-generation/dist/index';
```

Comment:
I changed all of these imports - is there a reason we had a relative link into node_modules previously? based on my understanding these should have 0 effect on the code (and I haven't been able to find a difference with manual testing), so I'm curious.

Reply:
Yeah I suspect it had something to do with getting things to work w/the rollup build for wallet generator, but can't remember exactly what was up.  Regardless: hacky relative link, begone!